roq-frontmatter/deployment/src/main/java/io/quarkiverse/roq/frontmatter/deployment/scan/RoqFrontMatterScanProcessor.java

```diff
@@ -163,8 +166,13 @@ private static Consumer<Path> addBuildItem(Path root, HashSet<RoqFrontMatterRawT
                     final String content = stripFrontMatter(fullContent);
                     String dateString = null;
                     if (fm.containsKey(DATE_KEY)) {
-                        ZonedDateTime date = ZonedDateTime.parse(fm.getString(DATE_KEY),
-                                DateTimeFormatter.ofPattern(config.dateFormat()));
+                        ZonedDateTime date = new DateTimeFormatterBuilder().appendPattern("yyyy-MM-dd[ HH:mm:ss Z]")
+                                .parseDefaulting(ChronoField.HOUR_OF_DAY, 12)
+                                .parseDefaulting(ChronoField.MINUTE_OF_HOUR, 0)
+                                .parseDefaulting(ChronoField.SECOND_OF_MINUTE, 0)
+                                .toFormatter()
+                                .withZone(ZoneId.of("UTC"))
```

Comment:
We should use the local timezone by default no?

Reply:
Yes, this only comes into play when there is no timzone set